### PR TITLE
D2D-050

### PR DESCRIPTION
**Pull request by Nibaron**
- checking email modal added.

image

### CODE REVIEW

File: assets/icons.tsx [hunk 1]
```diff
@@ -132,6 +132,25 @@ export const ArrowLeftCircleIcon = () => {
...
+    <svg
+      xmlns='http://www.w3.org/2000/svg'
+      width='21'
+      height='20'
+      viewBox='0 0 21 20'
+      fill='none'>
+      <path
+        d='M16.3332 9.99996H4.6665M4.6665 9.99996L10.4998 15.8333M4.6665 9.99996L10.4998 4.16663'
+        stroke='#131212'
+        stroke-width='1.66667'
```
Comment (tajnur007) [inline assets/icons.tsx:146]: React supports `strokeWidth` for `stroke-width`. Basically you have to discard the hypen from attribute name and the first character of second word will be in uppercase. e.g. `strokeLinecap`, `strokeLinejoin`.

File: assets/icons.tsx [hunk 2]
```diff
@@ -615,6 +634,35 @@ export const MapFlagIcon = () => {
...
+      xmlns='http://www.w3.org/2000/svg'>
+      <rect x='5.5' y='5' width='56' height='56' rx='28' fill='#DAD2FF' />
+      <rect
+        x='5.5'
+        y='5'
+        width='56'
+        height='56'
+        rx='28'
+        stroke='#EEEAFF'
+        stroke-width='10'
```
Comment (tajnur007) [inline assets/icons.tsx:653]: same here

File: assets/icons.tsx [hunk 3]
```diff
@@ -615,6 +634,35 @@ export const MapFlagIcon = () => {
...
+      />
+      <path
+        d='M21.8335 27.1666L31.3592 33.8346C32.1306 34.3746 32.5163 34.6446 32.9358 34.7492C33.3064 34.8415 33.694 34.8415 34.0645 34.7492C34.484 34.6446 34.8697 34.3746 35.6411 33.8346L45.1668 27.1666M27.4335 42.3333H39.5668C41.527 42.3333 42.5071 42.3333 43.2558 41.9518C43.9144 41.6163 44.4498 41.0808 44.7854 40.4223C45.1668 39.6736 45.1668 38.6935 45.1668 36.7333V29.2666C45.1668 27.3064 45.1668 26.3264 44.7854 25.5777C44.4498 24.9191 43.9144 24.3837 43.2558 24.0481C42.5071 23.6666 41.527 23.6666 39.5668 23.6666H27.4335C25.4733 23.6666 24.4932 23.6666 23.7445 24.0481C23.086 24.3837 22.5505 24.9191 22.215 25.5777C21.8335 26.3264 21.8335 27.3064 21.8335 29.2666V36.7333C21.8335 38.6935 21.8335 39.6736 22.215 40.4223C22.5505 41.0808 23.086 41.6163 23.7445 41.9518C24.4932 42.3333 25.4733 42.3333 27.4335 42.3333Z'
+        stroke='#5630FF'
+        stroke-width='2'
```
Comment (tajnur007) [inline assets/icons.tsx:658]: Same here and following

Comment (Nibaron) [reply assets/icons.tsx]: Done

**Review by tajnur007 — APPROVED** [review]
PR approved!

**Pull request merged**